apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -336,30 +361,47 @@ class ReviewTab extends Component {
 
     // channelId is not available on projects where the student has not edited the starter code.
     // comments cannot be made on projects in this tate.
-    const projectOwnerHasEditedCode = !!getStore().getState().pageConstants
+    const projectOwnerHasNotEditedCode = !getStore().getState().pageConstants
       .channelId;
 
-    if (projectOwnerHasEditedCode) {
+    if (projectOwnerHasNotEditedCode) {
+      return (
+        <div style={{...styles.reviewsContainer, ...styles.messageText}}>
+          {javalabMsg.noCodeReviewUntilStudentEditsCode()}
+        </div>
+      );
+    } else {
```

Comment:
nit: don't need to nest this inside of the `else` since we're returning above

Reply:
Seemed clearer to me to put it in else, but 🤷

Reply:
Yeah either works! It's just one less level of nesting but I'm totally fine either way

Reply:
I clearly was torn and did it one way one time, and one way the other 🤦